src/cli.jl

```diff
@@ -1,44 +1,57 @@
 """
-treeknit
+	treeknit
 
 # Arguments
 
-- `nwk1`: Newick file for first tree
-- `nwk2`: Newick file for second tree
+- `nwk_files`: Newick files (requires 2 or more trees - note an ARG currently cannot be created for more than 2 trees)
 
 # Options
 
 - `-o, --outdir <arg>`: directory for results; Example `-o=treeknit_results`
 - `-g, --gamma <arg>`: value of γ; Example `-g=2`
 - `--seq-lengths <arg>`: length of the sequences. Example: `--seq-length "1500 2000"`
 - `--n-mcmc-it <arg>`: number of MCMC iterations per leaf; default 25
+- `--rounds`: Number of times to run inference on input trees. If `rounds > 1` MCCs will be re-inferred using resolved trees from the last iteration. (default: `1` for 2 input trees, `2` for >2 input trees)
 
 # Flags
 
 - `--naive`: Naive inference (overrides `-g`).
 - `--no-likelihood`: Do not use branch length likelihood test to sort between different MCCs
 - `--no-resolve`: Do not attempt to resolve trees before inferring MCCs.
+- `--liberal-resolve`: Resolve output trees as much as possible using inferred MCCs, adding splits when order of coalescence and reassortment is unclear (coalescence is set at a time prior to reassortment)
+- `--parallel`: Run sequential multitree-TreeKnit with parallelization (only relevant for 4 or more trees)
+- `--final-no-resolve`: Do not resolve trees before inferring MCCs in final round of inference (default for more than 2 trees to prevent topological inconsistencies in output MCCs)
+- `--resolve-all-rounds`: Resolve trees before inferring MCCs in all rounds (default for 2 trees, overrides final-no-resolve)
 - `-v, --verbose`: verbosity
+- `--auspice-view`: return ouput files for auspice
 """
 @main function treeknit(
-	nwk1::AbstractString, nwk2::AbstractString;
+	nwk_file1::AbstractString, nwk_file2::AbstractString, nwk_files::AbstractString...;
 	# options
 	outdir::AbstractString = "treeknit_results",
 	gamma::Float64 = 2.,
-	seq_lengths::AbstractString = "1 1",
+	seq_lengths::AbstractString = join([string(i) for i in repeat([1], 2+ length(nwk_files))], " "),
 	n_mcmc_it::Int = 25,
+	rounds::Int = (length(nwk_files)>0) ? 2 : 1,
 	# flags
 	naive::Bool = false,
 	no_likelihood::Bool = false,
 	no_resolve::Bool = false,
+	liberal_resolve::Bool = false,
+	final_no_resolve::Bool = false,
```

Comment:
If I have more than 2 trees, this will get set to `true` anyway, whatever the input was, right? And if I have only two trees I would just use the `no_resolve` option. So maybe we could just remove this one from the argument list? 

By the way, if I want to resolve on the final round (for >2 trees) I have to set `resolve_all_rounds`  to `true`, right?

Reply:
yes - I have removed the option from the CLI for simplicity :-)